server/zoom/agent/action/action.py

```diff
@@ -118,7 +118,7 @@ def _callback(self):
                     self._mode != ApplicationMode.MANUAL:
                 self._log.info('Operational relevancy detected. '
                                'Triggering operation action.')
-                self._action_queue.append_unique(Task(self.name,
+                self._action_queue.append_unique(Task(self._op_action,
```

Comment:
this won't be very descriptive. `self._op_action` is function, not a string. Can you either change this to `self._op_action.__name__` or `"operational_action"`?